Components/Terrain/include/OgreTerrain.i

```diff
@@ -0,0 +1,47 @@
+%module(package="Ogre") Terrain
+%{
+/* Includes the header in the wrapper code */
+#include "Ogre.h"
+#include "OgreTerrain.h"
+#include "OgreTerrainAutoUpdateLod.h"
+#include "OgreTerrainGroup.h"
+#include "OgreTerrainPagedWorldSection.h"
```

Comment:
for now, you should stay away for the `*Paged*` stuff, as it depends on a separate component and thus complicates stuff.
Paging is not needed for "normal" Terrain usage.